tg/main.py

```diff
@@ -33,8 +33,7 @@ def run(tg: Telegram, stdscr: window) -> None:
     # run this function in thread?
     view = View(stdscr)
     model = Model(tg)
-    controller = Controller(model, view)
-    controller.tg = tg
+    controller = Controller(model, view, tg)
```

Comment:
not sure why do we need assign tg dynamically

Reply:
Correct, I fixed it in my patch as well